cmd/_integration-tests/README.md

```diff
@@ -1,40 +1,39 @@
 # integration-tests
 
 These tests run truss against definition files and tests if the generated code
-behaves. 
+behaves. Each test imports and uses service code generated by `truss`, but that code doesn't exist by default and must be generated before each test runs.
 
-# ./http
+# ./transport
 
-The `http` directory contains tests for the generated HTTP client libraries and
+The `transport` directory contains tests for the generated HTTP client libraries and
```

Comment:
Could you update this line to say that it "contains tests for the generated http and grpc transport libraries as well as standard Go 'net/http' requests.